contracts/token/ERC1155/utils/ERC1155HolderLean.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v5.0.0) (token/ERC1155/utils/ERC1155Holder.sol)
+
+pragma solidity ^0.8.20;
+
+import {IERC1155Receiver} from "@openzeppelin/contracts/interfaces/IERC1155Receiver.sol";
+
+/**
+ * @dev Version of {ERC1155Holder} that doesn't include {IERC165} detection.
```

Comment:
Do we want that? It goes against [the ERC-1155 specifications](https://eips.ethereum.org/EIPS/eip-1155#erc-1155-token-receiver)

> Smart contracts MUST implement the ERC-165 `supportsInterface` function and signify support for the `ERC1155TokenReceiver` interface to accept transfers. See “ERC1155TokenReceiver ERC-165 rules” for further detail.